client/src/app/project/[projectid]/searchComponents/SortFilterModal.tsx

```diff
@@ -0,0 +1,95 @@
+    }
+  };
+
+  //add location IDs of filtered data to be later passed on to map and search modules
+  const applyFilter = () => {
+    const tempIDs: string[] = [];
+
+    if (filterValues.length != 0) {
+      const filteredData = noteData.filter((note) => filterValues.includes(note.priority));
+      filteredData.forEach((note)=>{tempIDs.push(note.locationID)});
+      setLocationIDArray(tempIDs);
+    } else {
+      setLocationIDArray([]);
+    }
+    setLastFilterValues([...filterValues]);
+    setExpandModal(false);
+  };
+
+  //close modal and set filterValues to last set chosen by user
+  const cancelFilter = () => {
+    setExpandModal(false);
+    setFilterValues([...lastFilterValues]);
+  };
+
+  return (
+    <div>
+      <div className="flex flex-row justify-center items-center pb-4 text-2xl">
+        <div className="pr-4 text-lg font-bold">Searched Places</div>
+      </div>
+
+      <button onClick={() => {
+        setExpandModal(!expandModal);
+      }}>Sort/Filter</button>
+
+      {expandModal ? (
+        <div className="h-full flex justify-center items-center bg-gray-600/50 absolute inset-0">
+          <div className="flex justify-center flex-col items-center rounded-lg bg-Background_Lighter py-5 px-20 relative z-50">
+            <p className="font-medium text-2xl mb-4">Sort</p>
+            <ul className="flex flex-col space-y-2">
+              <li><button onClick={() => setSortValue("name")} className="w-full" autoFocus={sortValue === "name"}>by Name</button></li>
+              <li><button onClick={() => setSortValue("date")} className="w-full" autoFocus={sortValue === "date"}>by Date</button></li>
+              <li><button onClick={() => setSortValue("priority")} className="w-full" autoFocus={sortValue === "priority"}>by Priority</button></li>
+            </ul>
+
+            <p className="font-medium text-2xl my-4">Filter</p>
+            <ul className="flex flex-col space-y-2 mb-5">
+              <li>
+                <input type="checkbox" id="priorityLow" name="Low" value="Low" checked={filterValues.includes("Low")} onChange={() => handleFilterChange("Low")}/>
+                <label htmlFor="priorityLow"> Low Priority</label>
+              </li>
+              <li>
+                <input type="checkbox" id="priorityMedium" name="Medium" value="Medium" checked={filterValues.includes("Medium")} onChange={() => handleFilterChange("Medium")}/>
+                <label htmlFor="priorityMedium"> Medium Priority</label>
+              </li>
+              <li>
+                <input type="checkbox" id="priorityHigh" name="High" value="High" checked={filterValues.includes("High")} onChange={() => handleFilterChange("High")}/>
+                <label htmlFor="priorityHigh"> High Priority</label>
+              </li>
+            </ul>
+            <button onClick={cancelFilter} className="mb-2">Cancel</button>
+            <button onClick={applyFilter}>Apply Filters</button>
```

Comment:
To match the other designs, could this be updated to be more consistant to the below image: 

![image](https://github.com/AndrewLin95/next_destination_tracker/assets/107276941/e458bb6b-213a-4961-a77d-226e2e17338c)

Refer to: 
client > src > app > project > [projectid] > components > ProjectProfileDialog - Lines 154 - 168

```
          <div className="flex flex-row justify-end mt-2">
            <button
              type="submit"
              className={`${FORM_SUBMIT_BUTTON} h-10 w-32 mr-2`}
            >
              Submit
            </button>
            <button
              type="button"
              className={`${FORM_CANCEL_BUTTON} h-10 w-24 bg-SecondaryButton/80`}
              onClick={() => setProjectSettingsToggle(false)}
            >
              Cancel
            </button>
          </div>
```

Reply:
Most companies have style guidelines that are pretty strict (though it's kinda hard for you to know for this project unfortunately ><). I remember one of my PRs were sent back because a div was off by a few pixels lol 😭

Reply:
Damn by a few pixels?? I changed it so it uses the FORM_CANCEL_BUTTON and such but there may be some pixel differences :P

Reply:
I'm not that picky :p